solutions/swb-app/src/dataSetRoutes.ts

```diff
@@ -66,7 +64,7 @@ export function setUpDSRoutes(router: Router, dataSetService: DataSetPlugin): vo
 
   // import new prefix (assumes S3 bucket and path exist already)
   router.post(
-    '/datasets/import',
+    '/projects/:projectsId/datasets/import',
     wrapAsync(async (req: Request, res: Response) => {
       const validatedRequest = validateAndParse<CreateDataSetRequest>(req.body, CreateDataSetRequestParser);
       const dataSet = await dataSetService.importDataSet({
```

Comment:
How are you able to import a dateset without a project Id, but you need the project id to provision one? From a functional perspective, they only by whether a folder is created in the bucket.